repose-aggregator/components/filters/valkyrie-authorization-filter/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -436,7 +435,7 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
         extractDeviceIdFieldValue(value) flatMap { deviceIdFieldValue =>
           parseDeviceId(deviceIdFieldValue)
         } map { deviceId =>
-          devicePermissions.exists(_.device == deviceId.toInt)
+          devicePermissions.keySet.exists(deviceId.toInt.==)
```

Comment:
Man do i not like referencing the operator functions. it works just as well as `_ == deviceId.toInt` doesn't it? Because that seems like it's a little clearer to me.

Reply:
It does work just as well. I kind of liked the reference because I don't have to use a wildcard, but I don't actually have a strong opinion one way or the other.

I'll change it for you.

Reply:
And, in fact, that is equivalent to:
`.contains(deviceId.toInt)`